spec/helpers/reporters.js

```diff
@@ -0,0 +1,8 @@
+const SpecReporter = require('jasmine-spec-reporter').SpecReporter;
```

Comment:
There are semi-colons in this file, should they be removed to match other files?

Reply:
Done